common/types/activation.go

```diff
@@ -298,15 +298,26 @@ func (atx *ActivationTx) Verify(baseTickHeight, tickCount uint64) (*VerifiedActi
 	return vAtx, nil
 }
 
+// Merkle proof proving that a given leaf is included in the root of merkle tree.
+type MerkleProof struct {
+	Root      Hash32
+	Nodes     []Hash32 `scale:"max=32"`
```

Comment:
What does `Nodes` contain? Only the nodes on the path to the leaf?

Reply:
Yes. The blue nodes in https://medium.com/crypto-0-nite/merkle-proofs-explained-6dd429623dc5